packages/pds/src/db/db.ts

```diff
@@ -0,0 +1,113 @@
+import assert from 'assert'
+import {
+  Kysely,
+  SqliteDialect,
+  KyselyPlugin,
+  PluginTransformQueryArgs,
+  PluginTransformResultArgs,
+  RootOperationNode,
+  QueryResult,
+  UnknownRow,
+} from 'kysely'
+import SqliteDB from 'better-sqlite3'
+
+const DEFAULT_PRAGMAS = {
+  journal_mode: 'WAL',
+  busy_timeout: '5000',
+}
```

Comment:
We might consider `strict: 'ON'` here too https://sqlite.org/src/wiki/StrictMode

Reply:
oh yup good call 👍